package.json

```diff
@@ -2,7 +2,7 @@
   "name": "undp-radar-app",
   "version": "0.1.0",
   "private": true,
-  "homepage": "https://sdg-ai-lab.github.io/Digital_Technologies_Radar",
+  "homepage": "/Digital_Technologies_Radar",
```

Comment:
Better be generic as possible

Reply:
got it now. @VioletteHeron this part changed, just a reminder :)